larq/models.py

```diff
@@ -4,6 +4,7 @@
 
 import numpy as np
 import tensorflow as tf
+from packaging import version
```

Comment:
This is currently only a dev dependency, we'd need to add this to the proper install requirements: https://github.com/larq/larq/blob/43091316fff791b03f0eceabb4a75dc088ef41a6/setup.py#L34